pants.toml

```diff
@@ -14,8 +14,17 @@ backend_packages.add = [
   "pants.backend.python.typecheck.mypy",
   "pants.backend.python.mixed_interpreter_constraints",
   "internal_plugins.releases",
+  "toolchain.pants.auth",
+  "toolchain.pants.buildsense",
+  "toolchain.pants.common",
 ]
 
+plugins = [
+  "toolchain.pants.buildsense.plugin==0.1.0",
+]
+
+streaming_workunits_report_interval = 1
```

Comment:
This would probably make a better default than the current value. If you need to re-run this through CI, consider changing the default instead.

Reply:
I'll have a follow up PR to change the default for this and for run tracker options so we can remove those from pants.toml